iceoryx_examples/iceperf/main_follower.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "iceperf_follower.hpp"
+
+#include "iceoryx_utils/cxx/convert.hpp"
+#include "iceoryx_utils/platform/getopt.hpp"
+
+#include <iostream>
+
+int main(int argc, char* argv[])
+{
+    constexpr option longOptions[] = {
+        {"help", no_argument, nullptr, 'h'}, {"moo", required_argument, nullptr, 'm'}, {nullptr, 0, nullptr, 0}};
```

Comment:
Very nice ... But please listen carefully to this: https://www.youtube.com/watch?v=eK4Iz6K2xHQ